File: works/tmp/17388.js
let fs = require('fs');
// let input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');
let input = fs.readFileSync(0).toString().trim().split(' ');
// let input = fs.readFileSync('input.txt').toString().split(' ');

var [a, b, c] = input.map(Number) 
if(a + b + c >= 100) console.log("OK")
else 
{
    t = Math.min(a, b, c) 
    if(a == t) console.log("Soongsil")
    else if(b == t) console.log("Korea")
    else console.log("Hanyang")
}

File: works/tmp/17388.dart
import 'dart:io';
import 'dart:math';

void main()
{
    var str = stdin.readLineSync()!;
    var v1 = str.split(' ').map((o) => int.parse(o));
    int a = v1.elementAt(0), b = v1.elementAt(1), c = v1.elementAt(2);

    if(a + b + c >= 100) print("OK");
    else
    {
        int t = min(min(a, b), c);
        if(a == t) print("Soongsil");
        else if(b == t) print("Korea");
        else print("Hanyang");
    }
}